src/main/java/main/view/BaseServlet.java

```diff
@@ -139,15 +148,22 @@ protected void setErrorHeader(@NotNull HttpServletResponse resp, String errorMes
     }
 
     @Nullable
-    private String getSessionId(@NotNull HttpServletRequest req){
+    private String getSessionId(@NotNull HttpServletRequest req) throws AqualityException {
         String header = req.getHeader("Authorization");
         if(header != null){
+            validateAuthHeader(header);
             String[] strings = header.split(" ");
             return strings[1];
         }
         return null;
     }
 
+    private void validateAuthHeader(String header) throws AqualityException {
+        if(!header.toLowerCase().startsWith("basic ")){
```

Comment:
maybe we have to add toLowerCase to make "Basic" not to be case sensitive?

Reply:
fixed